src/utils.py

```diff
@@ -134,3 +135,18 @@ def ip_from_iface(subnet: str) -> Optional[str]:
 
     except AddrFormatError:
         return None
+
+
+def get_iface_ip_address(iface: str) -> Optional[str]:
+    """Get the UE IP address.
+
+    Args:
+        iface: The interface name.
+
+    Returns:
+        str: UE IP address
+    """
```

Comment:
I think the name of the function and the docstring don't really match.
Could you please change the docstring so the function is more general?
```suggestion
    """Get the IP address of the given interface.

    Args:
        iface: The interface name.

    Returns:
        str: IP address
    """
```

Reply:
Done. Copilot doing copilot things 😅